internal/pkg/configuration/configuration.go

```diff
@@ -62,6 +65,43 @@ func Generate(ctx context.Context, in *machine.GenerateConfigurationRequest) (re
 			taloscfgBytes []byte
 		)
 
+		if machineType != v1alpha1machine.TypeInit {
```

Comment:
I think we should phase out notion of `Init` node altogether

if we are running `machined` (vs. maintenance service), any config generated should use existing keys, not generate new ones

Reply:
ok, makes sense